tables/subject_plasma/duan22-plasma.csv

```diff
@@ -0,0 +1,10 @@
+﻿ref_name,subject_name,rx_name,collection_date_cmp,collection_date,location,cumulative_group,section
+Duan22,Wuhan_Hu_1_CP_BBIBP_CorV_1dose,Wuhan_Hu_1_CP_BBIBP_CorV_before_first_dose,~,2021-05-01,China,Wuhan_Hu_1_CP_BBIBP_CorV_1dose,NULL

```

Comment:
@KaimingTao I forget if I'm supposed to exclude observations that are without vaccinations/mAb; there are post-infection-pre-vaccination observations here and I'm not sure if they should be included or not

Reply:
@zachandfox it's ok, any plasma sample after infection should be included. We just exclude any plasma sample without infection or without vaccination.